src/mesh.cpp

```diff
@@ -8,6 +8,10 @@
 
 #include <set>
 
+#ifdef __OpenBSD__
+#define rand() arc4random()
```

Comment:
It seems [`arc4random`](https://man.openbsd.org/arc4random) is a "better" `rand` on OpenBSD. But why change to it? SolveSpace uses random numbers for completely "benign" things unrelated to security and I doubt it the quality of the PRNG matters at all.

Reply:
Actually `rand()` in OpenBSD is already some kind of wrapper to `arc4random` so really this PR is just to silence the compiler warning. I understand this might be unnecessary and would be okay leaving it out.